packages/lib/src/utils/errors/DeployError.ts

```diff
@@ -0,0 +1,8 @@
+export class DeployError extends Error {
+  constructor(error:Error, props:any) {
+    super(error.message);
+    this.stack = error.stack;
+    this.name = 'DeployError';
+    Object.keys(props).forEach(( prop:string ) => this[prop] = props[prop]);
```

Comment:
nit: I don't see any reason for adding spaces between the arrow function's params, as we are not adding them in regular functions.
```suggestion
    Object.keys(props).forEach((prop: string) => this[prop] = props[prop]);
```

Reply:
Agreed! Can you treat is as the above comments about TSLint? There is currently a recommended rule being enforced that forces () always on arrow functions which is kind of a pain.